source/MRMesh/MRLaplacian.h

```diff
@@ -40,7 +40,7 @@ class Laplacian
     Laplacian( Mesh & mesh ) : mesh_( mesh ) { }
     // initialize Laplacian for the region being deformed, here region properties are remembered and precomputed;
     // \param freeVerts must not include all vertices of a mesh connected component
-    MRMESH_API void init( const VertBitSet & freeVerts, EdgeWeights weights, RememberShape rem = RememberShape::Yes );
+    MRMESH_API void init( const VertBitSet & freeVerts, EdgeWeights weights, RememberShape rem = Laplacian::RememberShape::Yes );
```

Comment:
why is it necessary?

Reply:
A defect in my code generator. I can't find a way to make Clang print default arguments with full qualifiers. I've added `using namespace`s in the generated code, so we don't need to spell namespaces, but we still must spell class names.

Reply:
It is ok, but please put a comment in the code for someone not to return it back